src/components/utils/jwt.js

```diff
@@ -0,0 +1,12 @@
+const parseJwt = (token) => {
+  const base64Url = token.split('.')[1]
+  const base64 = base64Url.replace('-', '+').replace('_', '/')
+  return JSON.parse(atob(base64))
+}
+
+export const jwtExpired = (token) => {
+  if (!token) return true
```

Comment:
I would rather we did this check outside this function because the jwtExpired should not be responsible for whether or not the payload contained a token.